### PR TITLE
add DeepSpeed docker image

### PR DESCRIPTION
**Pull request by lanking520**
## Description ##

Add DeepSpeed docker image for large model supporting

### CODE REVIEW

File: serving/docker/deepspeed.Dockerfile [hunk 1]
```diff
@@ -0,0 +1,39 @@
+FROM nvidia/cuda:11.3.0-cudnn8-devel-ubuntu20.04
```
Comment (frankfliu) [inline serving/docker/deepspeed.Dockerfile:1]: Let name it cu113.Dockerfile, this docker can be a generic GPU docker

Comment (frankfliu) [reply serving/docker/deepspeed.Dockerfile:1]: And the base docker image should be parameterized as well

Comment (lanking520) [reply serving/docker/deepspeed.Dockerfile:1]: This is specialized for pytorch, transformers and deepspeed. For GPU generic docker, we should not install anything

Comment (frankfliu) [inline serving/docker/deepspeed.Dockerfile:1]:
```suggestion
ARG version=11.3.0-cudnn8-devel-ubuntu20.04
FROM nvidia/cuda:$version
```

**Pull request merged**